### ISSUE TITLE
Wings/buildings both as parents

### ISSUE THREAD

**Issue by rkdarst** [usefulaaltomap/usefulaaltomap#40]
Check some of the units within A Grid.  They are listed as being contained in both the wing and the building.  This makes contained/contains part of/contained in/in is a bit wrong and needs some sort of fixing.

**Comment by josephguillaume** [comment]
I was wondering about contains for the main building - the note says to pick the wing first, but there's a long list of rooms in the building, and none in the wing.
It seems like it would be useful for the building to have a nested structure which groups the rooms by wing (if you want to go straight there), and for the wing to have all the rooms repeated again (if you literally follow the note/search directly for the wing)?

**Comment by rkdarst** [comment]
Yeah, the nested structure would be best and current setup is mostly historical.  I didn't think about wings at first, and it's slowly grown and now there are inconsistencies.  I basically need to sit down and fix it - and hope it doesn't become unmanageable yet.
